### PR TITLE
Migrate split.dart to null safety.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/lib/src/shared/split.dart [hunk 1]
```diff
@@ -89,7 +86,7 @@ class Split extends StatefulWidget {
 }
 
 class _SplitState extends State<Split> {
-  List<double> fractions;
+  late List<double> fractions;
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/shared/split.dart:89]: can this be late final?

**Review by kenzieschmoll — APPROVED** [review]
LGTM with one comment

File: packages/devtools_app/lib/src/shared/split.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/shared/split.dart:89]: done

**Comment by polina-c** [comment]
tested

**Pull request merged**